static/main.js

```diff
@@ -32,9 +40,27 @@ function endPos() {
 	ctx.beginPath();
 
 	// send info using websocket
```

Comment:
This comment doesn't look like it's at the right place.

Reply:
Removed comment